core/src/main/java/org/apache/calcite/sql/dialect/SparkSqlDialect.java

```diff
@@ -982,4 +983,13 @@ private String removeDotFromAMAndPM(SqlNode dateStringSqlNode) {
     }
     return dateString;
   }
+
+  @Override protected void unparseFormat(
+      final SqlWriter writer,
+      final SqlCall call, final int leftPrec, final int rightPrec) {
+    final SqlWriter.Frame formatFrame = writer.startFunCall("STRING");
+    List<SqlNode> operands = call.getOperandList();
```

Comment:
As "operands" is not repetitively used, why store as variable.
We can use - call.getOperandList().get(1).unparse(writer, leftPrec, rightPrec);

Reply:
Done